genomeGraph/tournamentTree.go

```diff
@@ -1,351 +0,0 @@
-package genomeGraph
-
-import (
-	"log"
-
```

Comment:
This was an improvement to the first development and is moved to seed.go:
- https://github.com/vertgenlab/gonomics/blob/1d145e8a8bc1529182def67895aeadffff5ca970/genomeGraph/seed.go#L21-L29